File: pat-static/src/main/java/com/rectanglel/patstatic/routes/BusRoute.java
package com.rectanglel.patstatic.routes;

import com.google.gson.annotations.Expose;

/**
 * TrueTime representation of bus routes
 * <p>
 * Created by epicstar on 3/4/17.
 * @author Jeremy Jao
 */
public class BusRoute {
    @Expose
    private String rt;

    @Expose
    private String rtnm;

    @Expose
    private String rtclr;

    @Expose
    private String rtdd;

    public String getRouteNumber() {
        return rt;
    }

    public String getRouteName() {
        return rtnm;
    }

    public String getRouteColor() {
        return rtclr;
    }

    public String getRouteDd() {
        return rtdd;
    }
}

File: pat-static/src/main/java/com/rectanglel/patstatic/routes/BusRoute.kt
package com.rectanglel.patstatic.routes

import com.google.gson.annotations.SerializedName

/**
 * TrueTime representation of bus routes
 *
 *
 * Created by epicstar on 3/4/17.
 * @author Jeremy Jao
 */
data class BusRoute (
    @SerializedName("rt")
    val number: String,

    @SerializedName("rtnm")
    val name: String,

    @SerializedName("rtclr")
    val color: String,

    @SerializedName("rtdd")
    val designator: String,

    @SerializedName("rtpidatafeed")
    val datafeed: String
)
